package_info.json

```diff
@@ -1,4 +1,4 @@
 {
     "package_name": "th2_grpc_common",
-    "package_version": "2.3.5"
+    "package_version": "2.3.6"
```

Comment:
It is the info for the python publication, isn't it?

Reply:
Yeah, but guess it should equal to `release_version` - at least in other projects it is